components/global/core/io.cellery.observability.api/src/main/java/io/cellery/observability/api/siddhi/SiddhiStoreQueryManager.java

```diff
@@ -54,8 +54,15 @@
             "define table K8sPodInfoTable (cell string, component string, name string, creationTimestamp long, " +
             "lastKnownAliveTimestamp long, nodeName string);";
 
+    private static final String K8S_COMPONENT_INFO_TABLE = "@Store(type=\"rdbms\", " +
+            "datasource=\"CELLERY_OBSERVABILITY_DB\")\n" +
+            "@PrimaryKey(\"cell\", \"component\")\n" +
+            "@purge(enable=\"false\")\n" +
+            "define table K8sComponentInfoTable(cell string, component string, creationTimestamp long," +
+            " lastKnownActiveTimestamp long, Ingress_Types string)";
```

Comment:
Please change "Ingress_Types" field to the currently used convention. (ingressTypes)